signcrypt_seal.go

```diff
@@ -69,6 +69,7 @@ func (sss *signcryptSealStream) signcryptBytes(b []byte) error {
 
 	nonce := nonceForChunkSigncryption(sss.numBlocks)
 
+	// TODO: Encode payload chunk.
```

Comment:
Ah, yes, that brings up another question I forgot to ask. Doesn't it have to be in the hash input? There's anonymous mode, right, where the signature input is ignored...